test/setup/iam.tf

```diff
@@ -20,7 +20,8 @@ locals {
     "roles/compute.admin",
     "roles/run.admin",
     "roles/iam.serviceAccountUser",
-    "roles/certificatemanager.owner"
+    "roles/certificatemanager.owner",
+    "roles/owner"
```

Comment:
This is high privilege role and we don't want this to be given to the service account. Please include more granular role if needed.